packages/engine/Source/Core/ITwin.js

```diff
@@ -0,0 +1,306 @@
+  };
+
+  // obtain export for specified export id
+  let url = `${ITwin.apiEndpoint}mesh-export/?iModelId=${iModelId}`;
+  if (defined(changesetId) && changesetId !== "") {
+    url += `&changesetId=${changesetId}`;
+  }
+
+  const response = await fetch(url, { headers });
+  if (!response.ok) {
+    const result = await response.json();
+    if (response.status === 401) {
+      throw new RuntimeError(
+        `Unauthorized, bad token, wrong scopes or headers bad. ${result.error.details[0].code}`,
+      );
+    } else if (response.status === 422) {
+      throw new RuntimeError(
+        `Unprocessable Entity:${result.error.code} ${result.error.message}`,
+      );
+    } else if (response.status === 429) {
+      throw new RuntimeError("Too many requests");
+    }
+    throw new RuntimeError(`Unknown request failure ${response.status}`);
+  }
+
+  /** @type {{exports: Export[]}} */
+  const result = await response.json();
+  return result;
+};
+
+/**
+ * Start the export process for the given iModel + changeset.
+ *
+ * @param {string} iModelId
+ * @param {string} changesetId
+ */
+ITwin.createExportForModelId = async function (iModelId, changesetId) {
+  if (!defined(ITwin.defaultAccessToken)) {
+    throw new DeveloperError("Must set ITwin.defaultAccessToken first");
+  }
+
+  changesetId = changesetId ?? "";
+
+  const requestOptions = {
+    method: "POST",
+    headers: {
+      Authorization: ITwin.defaultAccessToken,
+      Accept: "application/vnd.bentley.itwin-platform.v1+json",
+      "Content-Type": "application/json",
+    },
+    body: JSON.stringify({
+      iModelId,
+      changesetId,
+      exportType: "CESIUM",
+    }),
+  };
+
+  // initiate mesh export
+  const response = await fetch(
+    `https://api.bentley.com/mesh-export/`,
+    requestOptions,
+  );
+
+  if (!response.ok) {
+    const result = await response.json();
+    if (response.status === 401) {
+      console.error(
+        result.error.code,
+        result.error.message,
+        result.error.details,
+      );
+      throw new RuntimeError(
+        "Unauthorized, bad token, wrong scopes or headers bad",
+      );
+    } else if (response.status === 403) {
+      console.error(result.error.code, result.error.message);
+      throw new RuntimeError("Not allowed, forbidden");
+    } else if (response.status === 422) {
+      console.error(result.error.code, result.error.message);
+      console.error(result.error.details);
+      throw new RuntimeError("Unprocessable: Cannot create export job");
+    } else if (response.status === 429) {
+      throw new RuntimeError("Too many requests");
+    }
+
+    throw new RuntimeError(`Unknown request failure ${response.status}`);
+  }
+
+  /** @type {ExportResponse} */
+  const result = await response.json();
+  return result.export.id;
+};
+
+/**
+ * Delete the specified export
+ *
+ * TODO: I'm not sure if we want this or not. Might belong better as an APP level function
```

Comment:
You won't need it, but it is a good exercise to test the API

Reply:
We'll likely want to pair the API here down to the minimum necessary for visualization– Managing exports and other function is something that would not be unique to CesiumJS, so [to reduce the maintenance burden](https://github.com/CesiumGS/cesium/blob/main/Documentation/Contributors/CodingGuide/README.md#design) we can omit it.